### ISSUE TITLE
Empty page contents in Understandings

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2960]
Example: https://www.w3.org/WAI/WCAG21/Understanding/documenting-accessibility-support

Empty page contents should either be removed or (better) the page should have section headings (see [WCAG SC 2.4.10](https://www.w3.org/TR/WCAG21/#section-headings))

**Comment by alastc** [comment]
I'll combine the template issues for these docs into #2951
